frontend/src/app/containers/SignInPage/index.tsx

```diff
@@ -1,7 +1,91 @@
-import React from 'react';
-
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { Main } from 'utils/urls';
+import { useSignInSlice } from './slice';
+import * as apiActions from 'api/actions';
+import { selectSignIn } from './slice/selectors';
 export type Props = {};
 
 export default function SignInPage(props: Props) {
-  return <div data-testid="SignInPage">signin</div>;
+  const dispatch = useDispatch();
+  const { actions } = useSignInSlice();
+  const signInState = useSelector(selectSignIn);
+
+  const history = useHistory();
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+
+  const onSigninClicked = () => {
+    if (!email) {
+      alert('Please enter email');
+    } else if (!password) {
+      alert('Please enter password');
+    } else if (signInState.signInResponse.loading) {
+      alert('Still loading');
+    } else {
+      dispatch(apiActions.signin.request({ email, password }));
+    }
+  };
+
+  useEffect(() => {
+    if (!signInState.signInResponse.loading) {
+      if (signInState.signInResponse.data) {
+        history.push(Main()); // TODO : testing
+      }
+      if (signInState.signInResponse.error) {
+        alert('No User Info Received!');
+      }
+    }
+  });
```

Comment:
SignIn 시 return해주는 게 204 no content라서 data에 아무것도 안 들어있을 수도 있는데, 아래와 같이 하는 건 어떨까 합니다

```suggestion
  useEffect(() => {
    if (!signInState.signInResponse.loading) {
      if (signInState.signInResponse.error) {
        alert('No User Info Received!');
      } else {
        history.push(Main());
      }
    }
  });
```

Reply:
넵 반영했습니다 !